assets/index.js

```diff
@@ -7,19 +7,41 @@ import 'whatwg-fetch';
 var filterOpen = false;
 var previewOpen = false;
 
-function responsiveWireItem() {
+function isGrid() {
+    if ( $('.wire-articles__item').hasClass('wire-articles__item--grid')) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+
+function responsiveWireItem() {    
+
     if (filterOpen && !previewOpen) {
         document.getElementsByClassName('wire-column__main')[0].classList.add('wire-articles__one-side-pane');
         document.getElementsByClassName('wire-column__main')[0].classList.remove('wire-articles__two-side-panes');
+        if ( isGrid() ) {        
+            $('.wire-articles__item-wrap').removeClass().addClass('wire-articles__item-wrap col-sm-12 col-md-6 col-xl-4 col-xxl-3');
```

Comment:
is this something we should be doing in react?

Reply:
Yes I suppose so.